h5netcdf/core.py

```diff
@@ -338,6 +354,18 @@ def _create_child_variable(self, name, dimensions, dtype, data, fillvalue,
         else:
             h5name = name
 
+        # None in the shape indicates unlimited dimensions. Set the maxshape
+        # to the shape in that case and replace all None's with zero for the
+        # actual shape. Or if there is data replace by the actual data size.
+        # This will allow resizing the data sets later if required.
+        if None in shape:
+            kwargs['maxshape'] = shape
+            if data is None:
+                shape = tuple(_i if _i is not None else 0 for _i in shape)
```

Comment:
Can you please drop the `_` prefix here? I understand the value of not leaking the variable into the outer scope, but this is harder to read and not a problem on Python 3, anyways.

Reply:
Sure. It just a habit I picked up due to scoping issues. It is still an problem with for loop indices in Python 3 but I agree that it is less serious nowadays.